radixdlt/babylon-wallet-android
# [ABW-3847] Integrate sargon logic for security problems in Android wallet

## Description
read [this](https://github.com/radixdlt/sargon/pull/287) 

## How to test

1. Create several security problems (e.g. cloud backup off and backup not updated) 
2. Ensure the security warnings are correct


## PR submission checklist
- [X] I have tested several security problems.
